### PR TITLE
Check if we have enough space in the pool for a vdisk

### PR DESCRIPTION
**Pull request by muhamadazmy**
Also fixes a precision issue with the counters

### CODE REVIEW

File: pkg/storage.go [hunk 1]
```diff
@@ -119,7 +119,7 @@ type Usage struct {
 	Used uint64
 }
 
-// Filesystem represents a storage space that can be used as a filesystem
+// Filesystem represents a storage space that can be used as a filesystem`
```
Comment (LeeSmet) [inline pkg/storage.go]: Out of curiosity, what's the backtick doing here?

**Review by LeeSmet — APPROVED** [review]

**Review by zaibon — APPROVED** [review]
LGTM

**Comment by LeeSmet** [comment]
It seems you need to regenerate the stubs though as the interface changed

File: pkg/storage.go (hunk 1 above)
Comment (muhamadazmy) [reply pkg/storage.go]: no idea.

**Pull request merged**